### ISSUE TITLE
border in td doesn't doesn't show top border

### ISSUE THREAD

**Issue by r4cker** [twbs/bootstrap#39082]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I want to hightlight a cell in a table. I use the class "border border-success" but the top border doesn't show. It seems like the row border is overriding the border.

The quick fix was to change the border style to ridge  : 

```
<style>
thead, tbody, tfoot, tr, td, th {
            border-style: ridge;
}
</style>
```

### Reduced test cases

https://codepen.io/Ben-Pouy/pen/BavywyQ

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.2

**Comment by Parasuramragu** [comment]
all are created successfully and there is no error in this site 

we are all wises to to create a good web site on as ur own thinks that we can seen by now

**Comment by mdo** [comment]
Yeah, this is because we collapse borders, so you'll have to target the proper border. In this case, `border-danger` on the column's `th`.
